src/Compilers/VisualBasic/Test/Symbol/SymbolsTests/Source/BaseClassTests.vb

```diff
@@ -2434,6 +2434,22 @@ BC30296: Interface 'A(Of T)' cannot inherit from itself:
 ]]></errors>)
         End Sub
 
+        <Fact>
+        Public Sub Tuple_MissingNestedTypeArgument()
```

Comment:
Please add a test with an extends clause (with a class) instead of an implements clause (with an interface).